chain/client/src/client_actor.rs

```diff
@@ -569,13 +569,6 @@ impl ClientActor {
 
         debug!(target: "client", "Check announce account for {}, epoch start height: {}, {:?}", validator_signer.validator_id(), epoch_start_height, self.last_validator_announce_height);
 
-        if let Some(last_validator_announce_height) = self.last_validator_announce_height {
```

Comment:
remove `last_validator_announce_height` altogether, and the computation of `epoch_start_height` as well?

Reply:
done! 👌